readme/api/references/rest_api.md

```diff
@@ -126,7 +126,7 @@ To retrieve all the tags that start with `project-`: **GET /search?query=project
 
 # Item type IDs
 
-Item type IDs might be refered to in certain object you will retrieve from the API. This is the correspondance between name and ID:
+Item type IDs might be refereed to in certain object you will retrieve from the API. This is the correspondence between name and ID:
```

Comment:
Shouldn't this be `referred` and not `refereed`?

Reply:
yep